docs/_includes/toggle-button.html

```diff
@@ -0,0 +1,254 @@
+<button type="button" class="toggle-button" aria-pressed="false">
+    <span class="toggle-button__content">
+        <span class="toggle-button__title">Text Button Toggled Off</span>
+    </span>
+</button>
+
+<button type="button" class="toggle-button" aria-pressed="true">
+    <span class="toggle-button__content">
+        <span class="toggle-button__title">Text Button Toggled On</span>
+    </span>
+</button>
+    {% endhighlight %}
+
+    <h4>Toggle Button with Title and Subtitle</h4>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <button type="button" class="toggle-button" aria-pressed="false">
+                <span class="toggle-button__content">
+                    <span class="toggle-button__title">Text Title</span>
+                    <span class="toggle-button__subtitle">Text Subtitle</span>
+                </span>
+            </button>
+        </div>
+    </div>
+
+    {% highlight html %}
+<button type="button" class="toggle-button" aria-pressed="false">
+    <span class="toggle-button__content">
+        <span class="toggle-button__title">Text Title</span>
+        <span class="toggle-button__subtitle">Text Subtitle</span>
+    </span>
+</button>
+    {% endhighlight %}
+
+    <h4>Toggle Button with Icon, Title, and Subtitle</h4>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <button type="button" class="toggle-button" aria-pressed="false">
+                <span class="toggle-button__icon">
+                    <svg aria-hidden="true" class="icon icon--on-the-way-16" focusable="false" height="16" width="16">
+                        {% include symbol.html name="on-the-way-16" prefix="icon" %}
+                </svg>
+                </span>
+                <span class="toggle-button__content">
+                    <span class="toggle-button__title">Icon Button Title</span>
+                    <span class="toggle-button__subtitle">Icon Button Subtitle</span>
+                </span>
+            </button>
+        </div>
+    </div>
+
+    {% highlight html %}
+<button type="button" class="toggle-button" aria-pressed="false">
+    <span class="toggle-button__icon">
+        <svg aria-hidden="true" class="icon icon--on-the-way-16" focusable="false" height="16" width="16">
+            <use href="static/icons.svg#icon-on-the-way-16"></use>
+        </svg>
+    </span>
+    <span class="toggle-button__content">
+        <span class="toggle-button__title">Icon Button Title</span>
+        <span class="toggle-button__subtitle">Icon Button Subtitle</span>
+    </span>
+</button>
+    {% endhighlight %}
+
+    <h4>Toggle Button with Image, Title, and Subtitle</h4>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <button type="button" class="toggle-button" aria-pressed="false">
+                <span class="toggle-button__image">
+                    <svg aria-hidden="true" class="icon icon--visa-32-colored" focusable="false" height="3" width="3">
```

Comment:
I'd like to see more examples here with an actual image instead of an icon. With some preliminary testing on my end it doesn't seem like images are styled at all.

I would especially like to see examples with ultra-wide images, which may have to be restricted in size horizontally like in Figma:

<img width="353" alt="image" src="https://github.com/eBay/skin/assets/26027232/37f0cc46-eef3-4888-a391-06c7e3db1de3">

Reply:
Will do!

Reply:
I'm actually curious about what happens here with images that are taller than they are wide, like vertical phone pictures. Does the larger card become very tall, or should we crop the image to fit into the landscape frame?

Reply:
Unless I hear otherwise from DS, I'm going to operate under the specification of having the `img` simply fit within the available vertical space similar to how a really, really wide image with a ludicrous aspect ratio would do the same horizontally. Other options have fairly sizable negative side-effects (see offline discussion). We can couple this with guidance that it's best to use landscape images in these and DS can additionally even provide an aspect ratio in the guidance as well.